modules/llrt_child_process/src/lib.rs

```diff
@@ -351,6 +344,216 @@ impl<'js> ChildProcess<'js> {
         }
         Ok(instance)
     }
+
+    fn exec_file(
+        ctx: Ctx<'js>,
+        command: String,
+        args: Option<Vec<String>>,
+        child: IoResult<Child>,
+        cb: Option<Function<'js>>,
+    ) -> Result<Class<'js, Self>> {
+        let (kill_signal_tx, kill_signal_rx) = broadcast_channel::<Option<i32>>(1);
+
+        let instance = Self {
+            emitter: EventEmitter::new(),
+            command: command.clone(),
+            args: args.clone(),
+            pid: None,
+            kill_signal_tx: Some(kill_signal_tx),
+        };
+
+        let stdout_instance = DefaultReadableStream::new(ctx.clone())?;
+        let stderr_instance = DefaultReadableStream::new(ctx.clone())?;
+        let stdin_instance = DefaultWritableStream::new(ctx.clone())?;
+
+        let instance = Class::instance(ctx.clone(), instance)?;
+        let instance2 = instance.clone();
+        let instance3 = instance.clone();
+        let instance4 = instance.clone();
+
+        instance.set("stderr", stderr_instance.clone())?;
+        instance.set("stdout", stdout_instance.clone())?;
+        instance.set("stdin", stdin_instance.clone())?;
+
+        match child {
+            Ok(mut child) => {
+                instance2.borrow_mut().pid = child.id();
+
+                if let Some(child_stdin) = child.stdin.take() {
+                    DefaultWritableStream::process(stdin_instance.clone(), &ctx, child_stdin)?;
+                };
+
+                let stdout_new: Vec<u8> = Vec::new();
+                let stderr_new: Vec<u8> = Vec::new();
+                let stdout_arc = Arc::new(Mutex::new(stdout_new));
+                let stderr_arc = Arc::new(Mutex::new(stderr_new));
+                let combined_stdout_buffer = Some(Arc::clone(&stdout_arc));
+                let combined_stderr_buffer = Some(Arc::clone(&stderr_arc));
```

Comment:
As far as I can see, we don't use None on this value anywhere.

Also depending on the usecase, it is usually fine to use a `RefCell` since the JS engine is single-threaded. As long as you don't hold the borrow across an await point or a callback to JS (anything that yields control back to the JS engine really).